runtime/caches/mod.ts

```diff
@@ -1,20 +1,34 @@
 import { withInstrumentation } from "./common.ts";
-import { compose } from "./compose.ts";
 import { caches as cachesKV } from "./denoKV.ts";
 import { caches as cachesProxy } from "./proxy.ts";
 import { caches as redisCache, redis } from "./redis.ts";
+import {
+  caches as cachesFileSystem,
+  isFileSystemAvailable,
+} from "./fileSystem.ts";
+import { caches as cachesS3, isS3Available } from "./s3.ts";
+import { createTieredCache } from "./tiered.ts";
 
-const DEFAULT_CACHE_ENGINE: CacheEngine = "CF_PROXY";
+const ENABLE_LOADER_CACHE = Deno.env.get("ENABLE_LOADER_CACHE") === "true";
+const DEFAULT_CACHE_ENGINE = ENABLE_LOADER_CACHE
+  ? "FILE_SYSTEM,S3"
```

Comment:
Instead, shouldn't we set the default to KV? Since this is going to be released in the new deco version?